go.mod

```diff
@@ -0,0 +1,5 @@
+module github.com/aprice/telnet
+
+go 1.16
```

Comment:
strictly this does not require 1.16 but I generally always put the latest major release there :-)